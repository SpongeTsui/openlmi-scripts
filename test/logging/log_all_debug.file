DEBUG: Running command "test".
DEBUG: Found registered command "test".
WARNING: Command "lmi.scripts.cmdtest.Show" is missing usage string. It will be inherited from parent command.
INFO: Connected to kvm-rhel7
DEBUG: One Ring to rule them all.
INFO: One Ring to find them.
WARNING: One Ring to bring them all
ERROR: and in the darkness bind them.
CRITICAL: In the land of Mordor where the Shadows lie.
