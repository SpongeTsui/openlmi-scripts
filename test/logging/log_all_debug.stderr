Running command "test".
Found registered command "test".
warning : Command "lmi.scripts.cmdtest.Show" is missing usage string. It will be inherited from parent command.
Connected to kvm-rhel7
One Ring to rule them all.
One Ring to find them.
warning : One Ring to bring them all
error   : and in the darkness bind them.
critical: In the land of Mordor where the Shadows lie.
